react-native/react/login/index.render.desktop.js

```diff
@@ -1,12 +1,69 @@
+/* @flow */
+/* eslint-disable react/prop-types */
+
 import React, {Component} from '../base-react'
+import {globalStyles} from '../styles/style-guide'
+import {resizeLoginForm} from '../local-debug'
+
+import {remote} from 'electron'
+
+import Carousel from '../util/carousel.desktop'
+
+import type {LoginRenderProps} from './index.render'
 
 export default class LoginRender extends Component {
-  render () {
+  props: LoginRenderProps;
+
+  window: ?any;
+  originalSize: ?{width: number, height: number};
+
+  componentWillMount () {
+    if (resizeLoginForm) {
+      this.window = remote.getCurrentWindow()
+      const [width, height] = this.window.getSize()
+      this.originalSize = {width, height}
+      this.window && this.window.setSize(styles.container.width, styles.container.height + 20 /* for frame */, true)
```

Comment:
can we use 
```
useContentSize Boolean - The width and height would be used as web page's size, which means the actual window's size will include window frame's size and be slightly larger. Default is false.
``` 
?

or at least move the 20 into the hacks section of the global-styles